src/core/commands.ts

```diff
@@ -73,7 +73,7 @@ export async function view(mode?: 'tab' | 'browser' | 'external' | vscode.Uri) {
     if (!pickedRootFile) {
         return
     }
-    return lw.viewer.view(lw.file.getPdfPath(pickedRootFile), typeof mode === 'string' ? mode : undefined)
+    return lw.viewer.view(vscode.Uri.parse(lw.file.getPdfPath(pickedRootFile)), typeof mode === 'string' ? mode : undefined)
```

Comment:
You are parsing a file path without scheme.

Reply:
This has nothing to do with .synctex.gz and should be fixed at the opportunity to modify the pdf.

Reply:
I don't get what you mean.